scripts/create_x509_chain.py

```diff
@@ -12,7 +12,8 @@ def create_custom_config():
     # Best options is to have the location of openssl config file in an env variable
     # The openssl config file extension could be "cfg" or "cnf"
 
-    config_path = os.getenv("OPENSSLCONFIG")
+    config_path = os.getenv("OPENSSL_CONF")
+    config_path = str(config_path)
```

Comment:
Can you have an environment variable that isn't a string?  It seems like converting config_path to a string should be a no-op.

Reply:
I deleted it but am not sure when this happened or why i have even done it

---
In reply to: [314835167](https://github.com/Azure/azure-iot-sdk-python-preview/pull/167#discussion_r314835167) [](ancestors = 314835167)